test/WebSites/ModelBindingWebSite/Models/Employee.cs

```diff
@@ -9,5 +11,14 @@ public class Employee : Person
         public string Department { get; set; }
 
         public string Location { get; set; }
+
+        [FromQuery(Name = "EmployeeId")]
+        public int Id { get; set; }
+
+        [FromRoute(Name = "EmployeeTaxId")]
+        public int TaxId { get; set; }
+
+        [FromForm(Name = "EmployeeSSN")]
+        public string SSN { get; set; }
```

Comment:
could make actions and tests more consistent if a `[ModelName]` attribute existed; current split discourages reuse and I suspect our customers will hit this limitation quickly.